pages/services/prometheus/0.1.0-2.3.1/connecting-clients/index.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
the entire sections needs to be updated based on what exists in the framework.

Reply:
Updated